mycroft/api/__init__.py

```diff
@@ -108,11 +108,11 @@ def send(self, params):
             data=data, json=json_body, timeout=(3.05, 15)
         )
         if response.status_code == 304:
-            LOG.debug('Etag matched. Nothing changed for: ' + params['path'])
+            # LOG.debug('Etag matched. Nothing changed for: ' + params['path'])
             response = self.etag_to_response[etag]
         elif 'ETag' in response.headers:
             etag = response.headers['ETag'].strip('"')
-            LOG.debug('Updating etag for: ' + params['path'])
+            # LOG.debug('Updating etag for: ' + params['path'])
```

Comment:
Can't we outright remove them? The etag system seems pretty stable now.

Reply:
I'm fine with removing.   This PR is partially for discussion points.